pkg/cluster/install.go

```diff
@@ -37,6 +37,7 @@ func (m *manager) AdminUpdate(ctx context.Context) error {
 		steps.Condition(m.apiServersReady, 30*time.Minute),
 		steps.Action(m.ensureBillingRecord), // belt and braces
 		steps.Action(m.fixSSH),
+		steps.Action(m.fixInfraID),        // Old clusters lacks infraID in the database. Which makes code prune to errors.
```

Comment:
```suggestion
		steps.Action(m.fixInfraID),        // Old clusters lacks infraID in the database. Which makes code prone to errors.
```

Reply:
Good catch!

![image](https://user-images.githubusercontent.com/509198/121890493-555a0380-cd12-11eb-98e6-84101b112138.png)

Reply:
Approved the PR before noticing that this hasn't been merged. I'm happy for this to be merged separately so this PR doesn't have to wait for e2es just because of a typo in a comment.